doc/implementation_details.rst

```diff
@@ -602,3 +614,21 @@ Run clang-format on staged files before committing::
 
 This will format the commit using clang-format. Examine and stage
 modified files before finalizing the commit.
+
+
+Workflow
+--------
+We have chosen to host the code on Github to ease the collaboration between
+users and different developers, and we take advantage of the standard
+Github workflow:
+
+* Open a Github issue on https://github.com/rtlabs-com/p-net/ for each
+  separate bug found.
+* Fork the repository to your own account on Github, and make a local
+  clone on your work station.
+* Create a branch named ``githubNUMBER_description``, where ``NUMBER`` is
```

Comment:
Will GH check the branch name to understand which issue it fixes? The usual method would be to mention the issue in the PR, and the submitter chooses whatever name they want for the branch. This workflow is probably already well documented on GH.

Reply:
Great, I will update the text.